### PR TITLE
Ss/auto merge

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: docs/install-and-customize.md [hunk 1]
```diff
@@ -98,7 +104,7 @@ In order to avoid port conflicts, by default you cannot collocate more than one
 
 # Installation Plan
 
-When the DC/OS Cassandra service is initially installed it will generate an installation plan as shown below. You can view, pause, and resume installation via the REST API. See [REST API authentication][#rest-auth] for information on how this request must be authenticated.
+When the DC/OS Cassandra service is initially installed it will generate an installation plan as shown below. You can view, pause, and resume installation via the REST API. See the REST API authentication section of the REST API Reference for information on how this request must be authenticated.
```
Comment (NimaVaziri) [inline docs/install-and-customize.md]: `DC/OS Apache Cassandra`?

File: docs/connecting-clients.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
 ---
 post_title: Connecting Clients
-menu_order: 70
+menu_order: 60
+feature_maturity: preview
+enterprise: 'yes'
 ---
 
-The only supported client for the DSOC Cassandra Service is the Datastax Java CQL Driver. Note that this means that Thrift RPC-based clients are not supported for use with this service and any legacy applications that use this communication mechanism are run at the user's risk.
+The only supported client for the DS/OS Cassandra Service is the Datastax Java CQL Driver. Note that this means that Thrift RPC-based clients are not supported for use with this service and any legacy applications that use this communication mechanism are run at the user's risk.
```
Comment (NimaVaziri) [inline docs/connecting-clients.md]: `DC/OS Apache Cassandra`

**Comment by NimaVaziri** [comment]
Besides the minor comments, this should be good to merge after a rebase.

File: docs/install-and-customize.md (hunk 1 above)
Comment (sascala) [inline docs/install-and-customize.md]: We only need to use "Apache" at first mention

File: docs/connecting-clients.md (hunk 2 above)
Comment (sascala) [inline docs/connecting-clients.md]: 👍

**Pull request merged**